src/components/Common/Navbar/Navbar.scss

```diff
@@ -0,0 +1,181 @@
+// Navbar.scss
+@import '../../../themes/mixins';
+@import '../../../themes/variables';
+
+.navbar-container {
```

Comment:
The structure of the SCSS could be organized more consistently. For example, some elements are nested under .navbar-container, while others are defined separately.